pkgs/development/python-modules/anthropic/default.nix

```
@@ -74,6 +72,11 @@ buildPythonPackage rec {
     ++ lib.optionals (pythonAtLeast "3.13") [
       # Fails on RuntimeWarning: coroutine method 'aclose' of 'AsyncStream._iter_events' was never awaited
       "test_multi_byte_character_multiple_chunks[async]"
+    ]
+    ++ lib.optionals lib.optionals (pythonAtLeast "3.13")
+    && (stdenvNoCC.hostPlatform.isDarwin) [
```

No parens needed here:
```suggestion
    && stdenvNoCC.hostPlatform.isDarwin [
```

---

Sorry, there were merge conflicts from an update and they left trash everywhere. About to fix.